Multisig/App/AppDelegate.swift

```diff
@@ -9,12 +9,15 @@
 import UIKit
 import SwiftUI
 import Firebase
+import Intercom
 
 class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         App.shared.firebaseConfig.setUp()
         App.shared.intercomConfig.setUp()
 
+        UIApplication.shared.registerForRemoteNotifications()
```

Comment:
Register on every app start. This does not trigger the messaging permission. The permission is till asked for when adding the first safe.